src/components/form/form.tsx

```diff
@@ -26,13 +28,36 @@ type formikOnChangeEventType =
 
 export const Form: React.FC = () => {
   // todo create hook for validating
+  const [buttonText, setButtonText] = useState("Submit");
   const [isFieldsHaveErrorMessages, setIsFieldsHaveErrorMessages] = useState({
     email: false,
     name: false,
     defineYourselfIndex: false,
     checkbox: false,
   });
 
+  const { response, isPending, error, reqHandler, resetStates } =
+    useRequest<ISendLetsTalkFormResponse>({
+      onSuccess: ({ isSucceeded }) => {
+        isSucceeded && toastGenerator({ isSuccess: true });
+      },
+      onFailure: (e) => {
+        const is429Error = (e as AxiosError)?.response?.status === 429;
```

Comment:
use meaningful naming. will someone else understand what this variable means?

Reply:
fixed